specifyweb/frontend/js_src/lib/localization/specifyNetwork.ts

```diff
@@ -83,83 +83,16 @@ export const specifyNetworkText = createDictionary({
   },
   mapDescription: {
     'en-us':
-      'This map shows all occurrences of this taxon from iDigBio and GBIF.',
-    'de-ch':
-      'Diese Karte zeigt alle Vorkommen dieses Taxons aus iDigBio und GBIF.',
-    'es-es': `
-      Este mapa muestra todas las ocurrencias de este taxón de iDigBio y GBIF.
-    `,
-    'fr-fr': `
-      Cette carte montre toutes les occurrences de ce taxon d'iDigBio et GBIF.
-    `,
-    'ru-ru':
-      'На этой карте показаны все вхождения этого таксона из iDigBio и GBIF.',
-    'uk-ua':
-      'Ця карта показує всі випадки появи цього таксону з iDigBio та GBIF.',
+      'This map shows occurrences of this taxon from the iDigBio and GBIF aggregators.',
   },
   iDigBioDescription: {
     'en-us': `
-      iDigBio points are represented as green dots on the map. Of those, the
-      occurrences published to iDigBio from the current collection are red.
-    `,
-    'de-ch': `
-      Bitte vergewissern Sie sich, dass Ihr Browser keine Pop-up-Fenster
-      blockiert, und versuchen Sie es erneut.
-    `,
-    'es-es': `
-      Los puntos iDigBio se representan como puntos verdes en el mapa. De esas,
-      las ocurrencias publicadas en iDigBio de la colección actual están en
-      rojo.
-    `,
-    'fr-fr': `
-      Les points iDigBio sont représentés par des points verts sur la carte.
-      Parmi ceux-ci, les occurrences publiées sur iDigBio à partir de la
-      collection actuelle sont rouges.
-    `,
-    'ru-ru': `
-      Точки iDigBio представлены на карте зелеными точками. Из них вхождения,
-      опубликованные в iDigBio из текущей коллекции, отмечены красным цветом.
-    `,
-    'uk-ua': `
-      Точки iDigBio представлені на карті зеленими крапками. З них опубліковані
-      в iDigBio випадки з поточної колекції червоні.
+      Occurrences recorded in iDigBio are shown as round green points, except for those from the current collection recorded in iDigBio that are round red points. Zooming in on the red points brings up a blue teardrop pin, clicking on the pin executes a locality query in Specify that shows all of the species in the current Specify database collected from that location.
```

Comment:
please avoid long lines - those cause horizontal scrolling in an editor
you can enable a feature in your text editor that automatically hard wraps line at 80 characters and you can also configure visual rulers - https://stackoverflow.com/questions/29968499/vertical-rulers-in-visual-studio-code